pandas/io/pytables.py

```diff
@@ -4712,12 +4712,13 @@ def is_transposed(self) -> bool:
     def get_object(cls, obj, transposed: bool):
         return obj
 
-    def write(self, obj, data_columns=None, **kwargs):
+    # error: Signature of "write" incompatible with supertype "Fixed"
+    def write(self, obj, data_columns=None, **kwargs) -> None:  # type: ignore[override]
```

Comment:
Can obj be added to the parent, or are there subclasses that don't take obj?

Reply:
I added obj for the parent methods: `obj` was only missing in "fake" abstract parent methods (not actually abstract, but raising NotImplemented).